src/directives/mmsSearch.js

```diff
@@ -697,6 +716,22 @@ function mmsSearch($window, $anchorScroll, CacheService, ElementService, Project
             };
         };
 
+        var combineRelatedViews = function(scope) {
+            scope.searchResults.forEach(function(element) {
+                var allRelatedDocuments = [];
+                element._relatedDocuments.forEach(function(relatedDoc) {
```

Comment:
need to make sure that _relatedDocuments isn't empty

Reply:
error:
TypeError: Cannot read property 'forEach' of undefined
    at http://localhost:9000/js/ve-mms.min.58274fd073311659.js:14463:43
    at Array.forEach (<anonymous>)
    at combineRelatedViews (http://localhost:9000/js/ve-mms.min.58274fd073311659.js:14461:33)
    at http://localhost:9000/js/ve-mms.min.58274fd073311659.js:14139:21
    at processQueue (http://localhost:9000/js/vendor.min.af769678f49b3e97.js:25907:28)

Reply:
Interesting. I've seen that the server return empty array for this field when there are no related documents. Guess it doesn't always do that. I add a precheck just in case when that happens.